slack/models/comms_channel.py

```diff
@@ -17,6 +19,13 @@ def create_comms_channel(self, incident):
         try:
             name = f"inc-{100+incident.pk}"
             channel_id = get_or_create_channel(name)
+
+            doc_url = urljoin(
```

Comment:
Can these be moved outside the try/except?  They're not related to the handling of the `SlackError` so worth having them outside.

Reply:
Good point. I've moved this to after the create channel try-catch.